### ISSUE TITLE
Fail to Load CLIP Model (CLIP-ViT-B-32)

### ISSUE THREAD

**Issue by debanjum** [UKPLab/sentence-transformers#1659]
### Intro
- Loading `CLIP-ViT-B-32` model fails with *No such file or directory* error for `fast_tokenizer_file`
- Seems related to https://github.com/huggingface/transformers/issues/15283
- Is anyone else facing this issue? Is there any known workaround or fix for this?

### Minimal Repro:
```sh
from sentence_transformers import SentenceTransformer
model = SentenceTransformer('sentence-transformers/clip-ViT-B-32')
```

### System Info:
- OS: MacOS Monterey, Ubuntu 22.04 (Jammy) (via Docker)
- python >= 3.9
- torch >= 1.11.0,
- torchvision >= 0.12.0,
- transformers >= 4.21.0,
- sentence-transformers >= 2.1.0,
- openai >= 0.20.0
- huggingface_hub >= 0.8.1

### Stacktrace

```python
.../.venv/lib/python3.9/site-packages/sentence_transformers/models/CLIPModel.py in __init__(self, model_name, processor_name)
     13
     14         self.model = transformers.CLIPModel.from_pretrained(model_name)
---> 15         self.processor = transformers.CLIPProcessor.from_pretrained(processor_name)
     16
     17     def __repr__(self):

.../.venv/lib/python3.9/site-packages/transformers/processing_utils.py in from_pretrained(cls, pretrained_model_name_or_path, **kwargs)
    184                 [`~tokenization_utils_base.PreTrainedTokenizer.from_pretrained`].
    185         """
--> 186         args = cls._get_arguments_from_pretrained(pretrained_model_name_or_path, **kwargs)
    187         return cls(*args)
    188

.../.venv/lib/python3.9/site-packages/transformers/tokenization_utils_base.py in from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs)
   1803                 logger.info(f"loading file {file_path} from cache at {resolved_vocab_files[file_id]}")
   1804
-> 1805         return cls._from_pretrained(
   1806             resolved_vocab_files,
   1807             pretrained_model_name_or_path,

.../.venv/lib/python3.9/site-packages/transformers/tokenization_utils_base.py in _from_pretrained(cls, resolved_vocab_files, pretrained_model_name_or_path, init_configuration, use_auth_token, cache_dir, *init_inputs, **kwargs)
   1948         # Instantiate tokenizer.
   1949         try:
-> 1950             tokenizer = cls(*init_inputs, **init_kwargs)
   1951         except OSError:
   1952             raise OSError(

.../.venv/lib/python3.9/site-packages/transformers/models/clip/tokenization_clip_fast.py in __init__(self, vocab_file, merges_file, tokenizer_file, unk_token, bos_token, eos_token, pad_token, **kwargs)
     87         **kwargs
     88     ):
---> 89         super().__init__(
     90             vocab_file,
     91             merges_file,

.../.venv/lib/python3.9/site-packages/transformers/tokenization_utils_fast.py in __init__(self, *args, **kwargs)
    108         elif fast_tokenizer_file is not None and not from_slow:
    109             # We have a serialization from tokenizers which let us directly build the backend
--> 110             fast_tokenizer = TokenizerFast.from_file(fast_tokenizer_file)
    111         elif slow_tokenizer is not None:
    112             # We need to convert a slow tokenizer to build the backend

Exception: No such file or directory (os error 2)
```

**Comment by debanjum** [comment]
TLDR: Investigated this further and resolved the issue by deleting the cached CLIP model on my device

### Investigation
- Put a breakpoint at the point in code where file not found error was being thrown
  ```python
   .../.venv/lib/python3.9/site-packages/transformers/tokenization_utils_fast.py in __init__(self, *args, **kwargs)
       108         elif fast_tokenizer_file is not None and not from_slow:
       109             # We have a serialization from tokenizers which let us directly build the backend
   --> 110             fast_tokenizer = TokenizerFast.from_file(fast_tokenizer_file)
  ```
- Saw that the `fast_tokenizer_file` was set to `/home/ukh-reimers/.cache/huggingface/transformers/7811def0c53be25ba790cb67ac785669b508a8d1cf8c912b8ac046c5f08aee68.c51a626c1dc62de865745e4ab7125f190823b458ac5ee9ce21b1d006512b3a53`
- Realized the issue was (partially?) with the older models being cached on my machine. 
- Fixing the path to look under my home directory instead of `ukh-reimers` did not find the `fast_tokenizer_file` either

### Fix
- Delete cached CLIP torch model and (optionally?) the cached huggingface fast_tokenizer_files
  ```shell
  rm -rf ~/.cache/huggingface/transformers/*
  rm -rf ~/.cache/torch/sentence_transformers/sentence-transformers_clip-ViT-B-32
  ```
